src/main/java/codeu/controller/ActivityServlet.java

```diff
@@ -1,27 +1,83 @@
 
-
 package codeu.controller;
 
+import codeu.model.data.Conversation;
+import codeu.model.data.Message;
 import java.io.IOException;
 
-import codeu.model.data.User;
+import codeu.model.store.basic.ConversationStore;
+import codeu.model.store.basic.MessageStore;
 import codeu.model.store.basic.UserStore;
+import java.util.List;
+import java.util.UUID;
 
 import javax.servlet.ServletException;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+/*Servlet class responsible for the activity feed page*/
 public class ActivityServlet extends HttpServlet {
+        
+    /*Store class that gives access to users*/
+	private UserStore userStore;
+    /*Store class that gives access to conversations*/
+	private ConversationStore conversationStore;
+	/** Store class that gives access to Messages. */
```

Comment:
Here you have '/** ' whereas above you use '/* '
Here are the difference https://stackoverflow.com/questions/29815636/and-in-java-comments/47787786

Prefer '/** ' to generate java doc.